docs/index.md

```diff
@@ -13,11 +13,14 @@ hero:
     - theme: alt
       text: Angular Example
       link: https://six-group.github.io/six-webcomponents/demo/angular
+    - theme: alt
+      text: Vue Example
+      link: https://six-group.github.io/six-webcomponents/demo/vue
 features:
   - title: Agnostic
     details:
-      Our Library has first class support for Angular and React right out of the box. Checkout the
-      example apps above!
+      Our Library has first class support for Angular, React and Vue right out of the box. Checkout
```

Comment:
I would refrain from stating that we have full support for Vue. I would split Vue outside and say "beta support for Vue"